src/bindings/python/src/pyopenvino/graph/passes/manager.cpp

```diff
@@ -141,4 +128,6 @@ void regclass_graph_passes_Manager(py::module m) {
             pass_manager = Manager()
             pass_manager.register_pass("Serialize", xml_path="example.xml", bin_path="example.bin", version="IR_V11")
     // )");
+
+    manager.def("register_pass", &ov::pass::Manager::register_pass_instance);
```

Comment:
Docs and py::args, you might also like to add `py::prepend` to it or just simply move it before to the top of the "register_pass" chain of functions. This will move it to the top of the overload resolution order.

Reply:
Done.